libs/product/src/models/bundled-product.ts

```diff
@@ -1,18 +1,12 @@
 import { DaffProduct } from './product';
 import { DaffBundledProductItem } from './bundled-product-item';
 
-export enum MagentoPriceViewEnum {
```

Comment:
Remove these types, we don't have a purpose for them in Daffodil yet.

Reply:
done